components/style/values/generics/grid.rs

```diff
@@ -701,3 +701,13 @@ pub enum GridTemplateComponent<L> {
     /// A `subgrid <line-name-list>?`
     Subgrid(LineNameList),
 }
+
+impl<L> GridTemplateComponent<L> {
+    /// Returns length of <track-size> list.
```

Comment:
Sorry for not being clearer.

I meant that I'd expect (given the impl) to mention `<track-list>` instead... But I guess the comment implies that `<track-list>` is a list of `<track-size>`s and you're returning the length of that, which is true... Maybe mention `the <track-list>s <track-size>` instead?

Also, seems a really generic name, given that there's also the `LineNameList` in the `Subgrid` variant... Perhaps this should be renamed to `track_list_len()` or something like that?